### PR TITLE
Documented that default container network trust level is publish

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic-product/issues/575.

@hmahmood can you please review? Thanks!

### CODE REVIEW

File: docs/user_doc/vic_app_dev/network_use_cases.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ To perform certain networking operations on containers, your Docker environment
 - For an overview of the networks that vSphere Integrated Containers Engine uses, see [Virtual Container Host Networking](../vic_vsphere_admin/networks.md) in *Install, Deploy, and Maintain the vSphere Integrated Containers Infrastructure*.
 - For examples of how to deploy VCHs with different network configurations, see the section in Advanced Examples of Deploying a VCH on [Networking Examples](../vic_vsphere_admin/vch_installer_examples.md#networking) in *Install, Deploy, and Maintain the vSphere Integrated Containers Infrastructure*.
 
+**NOTE**: The default level of trust on VCH container networks is `published`. As a consequence, if the vSphere administrator did not configure `--container-network-firewall` on the VCH, you must specify `-p 80` in `docker run` and `docker create` commands to publish port 80 on a container. Alternatively, the vSphere administrator can reconfigure the VCH to set [`--container-network-firewall`](../vic_vsphere_admin/configure_vch.md#containernet) to a different level.
```
Comment (anchal-agrawal) [inline docs/user_doc/vic_app_dev/network_use_cases.md]: Nit for consistency: reconfigure the VCH  -> configure the VCH

File: docs/user_doc/vic_vsphere_admin/vch_installer_options.md [hunk 2]
```diff
@@ -787,6 +787,8 @@ The `--container-network-firewall` option allows you to set the following levels
 
 <pre>--container-network-firewall <i>port_group_name</i>:<i>trust_level</i></pre>
 
+If you do not set `--container-network-firewall`, the default level of trust is `published`. As a consequence, if you do not set `--container-network-firewall`, container developers must specify `-p 80` in `docker run` and `docker create` commands to publish port 80 on a container. In regular Docker, then do not need to specify `-p` to publish port 80.
```
Comment (anchal-agrawal) [inline docs/user_doc/vic_vsphere_admin/vch_installer_options.md]: Typo: then do not need -> they do not need

Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/vch_installer_options.md]: Fixed

File: docs/user_doc/vic_app_dev/network_use_cases.md (hunk 1 above)
Comment (stuclem) [reply docs/user_doc/vic_app_dev/network_use_cases.md]: Fixed. Thanks @anchal-agrawal and @hmahmood !

**Pull request merged**